dev/com.ibm.ws.jaxws.ejb_fat/fat/src/com/ibm/ws/jaxws/ejb/suite/FATSuite.java

```diff
@@ -46,7 +47,7 @@
                 EJBWSInterceptorTest.class
 })
 public class FATSuite {
-    @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModification().andWith(new FeatureReplacementAction().addFeature("jaxws-2.3").removeFeature("jaxws-2.2").withID("jaxws-2.3"));
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly()).andWith(FeatureReplacementAction.EE8_FEATURES().removeFeature("jsp-2.2").removeFeature("jaxws-2.2").addFeature("jaxws-2.3").withID("jaxws-2.3")).andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("jaxws-2.3").addFeature("xmlwstest-3.0"));
```

Comment:
xmlwstest-3.0 add isn't needed since this fat suite doesn't use jaxwstest test bundle.

Reply:
jsp-2.2 isn't used in any of the server.xml files so it isn't needed in the removeFeatures list.